lib/ansible/runner.rb

```diff
@@ -2,15 +2,32 @@ module Ansible
   class Runner
     class << self
       def run(env_vars, extra_vars, playbook_path)
-        run_via_cli(env_vars, extra_vars, playbook_path)
+        run_via_cli(env_vars, extra_vars, :playbook_path => playbook_path)
+      end
+
+      def run_role(env_vars, extra_vars, role_name, roles_path:, role_skip_facts: true)
```

Comment:
For Galaxy roles, `roles_path` argument won't mean anything because they are installed globally AFAIK. We would probably want to make it optional.

Reply:
@agrare although we probably didn't want to install the roles in a global dir, because of the possible conflicts with other providers?

Reply:
Right we'll need to look into if we can install galaxy roles to directories other than a global location.  Ideally we would scope this to per-provider repos so nothing conflicts.